modules/programs/direnv.nix

```diff
@@ -121,8 +121,14 @@ in
   config = mkIf cfg.enable {
     home.packages = [ cfg.package ];
 
-    xdg.configFile."direnv/direnv.toml" = mkIf (cfg.config != { }) {
-      source = tomlFormat.generate "direnv-config" cfg.config;
+    xdg.configFile."direnv/direnv.toml" = mkIf (cfg.config != {} || cfg.silent) {
+      source = tomlFormat.generate "direnv-config" cfg.config
+        // (lib.mkIf cfg.silent {
```

Comment:
Should probably be checking the package version, since this was a recent change.

Reply:
Also I don't know how to check the package version here, is there a "version" property on cfg.package? Could you provide an example please?

Reply:
> > I just downgraded to direnv 2.35 and the "new" way of silencing it does still work, would you still prefer to check the direnv version?
> 
> That doesn't really make sense... they only added these config values in this release [direnv/direnv@v2.35.0...v2.36.0#diff-a59528653ec4d512d46856d5f6f241559e6839b4ca63d60e427d298b189003e9R63-R64](https://github.com/direnv/direnv/compare/v2.35.0...v2.36.0#diff-a59528653ec4d512d46856d5f6f241559e6839b4ca63d60e427d298b189003e9R63-R64)

Yeah, my bad, I didn't properly clear my environment variables..